modules/survey_accounts/php/addsurvey.class.inc

```diff
@@ -77,6 +77,19 @@ class AddSurvey extends \NDB_Form
               array('CandID' => $error);
         }
 
+        $projectID = $db->pselect(
+            "SELECT ProjectID FROM session
+            WHERE CandID=:v_CandID",
+            array(
+                'v_CandID' => $values['CandID']
+            )
+        );
+        if (!\NDB_Factory::singleton()->user()->hasProject(reset($projectID[0]))) {
```

Comment:
Can you explain your commit message "Special visit label case handled"? I'm not sure why the code was changed here.